docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -81,6 +81,15 @@ To learn more about the ``ansible-galaxy`` command-line tool, see the :ref:`ansi
 
 .. _creating_collection_skeletons:
 
+Creating a collection using the ansible-creator tool
+====================================================
+
+The `ansible-creator <https://github.com/ansible/ansible-creator/>`_ tool is designed for effortlessly scaffolding your Ansible content.
+After `installing <https://ansible.readthedocs.io/projects/creator/installing/#installation>`_ ``ansible-creator`` you have two options to interact with it to create a foundational structure for the project:
```

Comment:
```suggestion
After `installing <https://ansible.readthedocs.io/projects/creator/installing/#installation>`_ ``ansible-creator`` you can initialize a project in one of the following ways:
```

Reply:
This just seems a bit less wordy to me tbh.

Reply:
Yes, and an important point - Longer sentences are harder to understand, especially for those for whom English is not their native language.  Also worth mentioning that avoiding 'it' is also a good thing, to avoid possible confusion on what 'it' is referring to.